app/js/components/App.jsx

```diff
@@ -7,15 +7,43 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { PatientHeader } from '@openmrs/react-components';
 
+import patientAction from '../actions/patientAction';
 import LabOrdersList from './LabOrdersList';
 
-export default class App extends React.Component {
+export class App extends React.Component {
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch(patientAction.getPatientRecord('da5e210f-a3c4-4c49-80f2-e2e5386db8ad'));
```

Comment:
I am hardcoding the Patient ID for now, till we can get it from the URL

Reply:
Cool, this is an existing patient on HUM-CI?